godtools/App/Features/Onboarding/Views/OnboardingQuickStartCell.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  OnboardingQuickStartCell.swift
+//  godtools
+//
+//  Created by Robert Eldredge on 11/8/21.
+//  Copyright © 2021 Cru. All rights reserved.
+//
+
+import UIKit
+
+class OnboardingQuickStartCell: UITableViewCell {
+    
+    static let nibName: String = "OnboardingQuickStartCell"
+    static let reuseIdentifier: String = "OnboardingQuickStartCellReuseIdentifier"
+        
+    private var item: OnboardingQuickStartItem?
```

Comment:
Hey @reldredge71 I see you added a ViewModel for this cell, but it hasn't been added to the Xcode project, I do see it in Finder however.  Can you use a ViewModel here instead of accessing the model directly to keep consistent with the View/ViewModel pattern.